.goreleaser.yml

```diff
@@ -46,7 +46,8 @@ signs:
       - "--detach-sign"
       - "${artifact}"
 release:
-  draft: true
+  name_template: "{{.Version}}"
+  prerelease: auto
```

Comment:
I tested the goreleaser changes in my [fork](https://github.com/andreaangiolillo/terraform-provider-mongodbatlas/releases). The `prerelease: auto` correctly created a pre-release when using the tag `vX.Y.Z-pre`